.gitignore

```diff
@@ -0,0 +1,4 @@
+db/*
+lib/*
+node_modules/*
+npm-debug.log
```

Comment:
You should probably look at your other gitignores and add stuff now. I'm sure DS_STORE will be a pain, for example.